src/browser/chat/ChatRoomPage.js

```diff
@@ -0,0 +1,100 @@
+/* @flow */
+import React, { Component } from 'react';
+import { PageHeader, Title, View, Input, Form } from '../app/components';
+import { connect } from 'react-redux';
+import Chat from './ChatPage';
+import { joinRoom, sendMessage, loadMessages, kickUserFromRoom } from '../../common/lib/redux-firebase/actions';
+
+const ChatRoomMessage = ({ sender, content }) => (
+  <View>
+    {sender}
+    : {content}
+  </View>
+);
+
+class ChatRoomMessages extends Component {
+  componentWillMount() {
+    this.props.loadMessages(this.props.chatRoom.id);
+  }
+  onInputKeyDown = (event) => {
```

Comment:
Ok ! I just saw that it was done that way here : https://github.com/nmalzieu/este-chat/blob/master/src/browser/todos/NewTodo.js#L12 😄